aiidalab_widgets_base/viewers.py

```diff
@@ -1136,7 +1136,14 @@ def _observe_supercell(self, _=None):
             self.set_trait(
                 "displayed_structure", None
             )  # To make sure the structure is always updated.
-            self.set_trait("displayed_structure", self.structure.repeat(self.supercell))
+            # because nglview loads the structure through save and load a temp "pdb" file
+            # thus change the cell and positions into standard form.
```

Comment:
Would it possible to add a sentence what the "standard form" actually is?
(I know this is probably basic knowledge, but not for people outside of solid state).

Reply:
I added the description on "standard form" and refined the comment for clarity.